postman/camp-trello-clone.postman_attachments.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
use PORT variable in postman URL: http://localhost:{{PORT}}
we use 8800 not 8085